### PR TITLE
Added some tests, slight refactor of Account.send_to_address

### PR DESCRIPTION
**Pull request by mmodelski**
Added a simple test for Account.calculate_total_balance()

### CODE REVIEW

File: accounts/tests.py [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+
+class AccountModelTests(TestCase):
 
-# Create your tests here.
+    def test_calculate_total_balance_smoke(self):
+        btc = Asset(ticker="BTC", atomic_unit="satoshi", base_unit="bitcoin")
+        btc.save()
+        account1 = Account(id=1, created_at=timezone.now(), balance=10, asset=btc)
+        account2 = Account(id=2, created_at=timezone.now(), balance=20, asset=btc)
+        account1.save()
```
Comment (spotty-banana) [inline accounts/tests.py]: Balances should always match the transaction objects. Initially set accounts to 0, and then create transaction objects.

File: accounts/tests.py [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+        account2.save()
+        Transaction.objects.create(created_at=timezone.now(), asset=btc,
```
Comment (spotty-banana) [inline accounts/tests.py]: Better here maybe to use the wallet functions for transfering assets

Comment (spotty-banana) [reply accounts/tests.py]: Though this depends on the point of the test - if the point is to only test the calculate_total_balance function and fake everything else then yes.

Personally not a big fan of unit testing, lots of work and "false code" for little benefit...

File: accounts/models.py [hunk 3]
```diff
@@ -253,25 +253,8 @@ def send_to_address(self, amount, to_address):
...
-                    # TODO: log this error somewhere
-                    pass
-
-                return tx
-
-            elif rows_updated > 1:
-                raise Exception("multiple rows were updated")
-
-            return 
+            self.send_to_account(asset_address.account, amount, TxType.TRANSFER)
```
Comment (spotty-banana) [inline accounts/models.py]: The problem with this is that it doesn't set the to_internal_address, which is information we would like to preserve.

Comment (mmodelski) [reply accounts/models.py]: Added additional parameter with default value.

File: accounts/tests.py (hunk 2 above)
Comment (mmodelski) [reply accounts/tests.py]: I guess for this test it is the simplest to fake the balance.

**Review by spotty-banana — COMMENTED** [review]
Looks good now

**Pull request merged**